src/Compilers/Core/Portable/Operations/OperationInterfaces.xml

```diff
@@ -3238,4 +3238,39 @@
       </Comments>
     </Property>
   </Node>
+  <Node Name="IImplicitIndexerReferenceOperation" Base="IOperation" ChildrenOrder="Instance,Argument" HasType="true">
+    <Comments>
+      <summary>
+        Represents a reference to an implicit System.Index or System.Range indexer over non-array type.
+        <para>
+          Current usage:
+          (1) C# implicit System.Index or System.Range indexer reference expression.
+        </para>
+      </summary>
+    </Comments>
+    <Property Name="Instance" Type="IOperation">
+      <Comments>
+        <summary>Instance of the type to be indexed.</summary>
+      </Comments>
+    </Property>
+    <Property Name="Argument" Type="IOperation">
+      <Comments>
+        <summary>System.Index or System.Range value.</summary>
+      </Comments>
+    </Property>
+    <Property Name="LengthSymbol" Type="ISymbol">
+      <Comments>
+        <summary>
+          The <c>Length</c> or <c>Count</c> property that can be used to fetch the length value.
```

Comment:
>can [](http://example.com/codeflow?start=58&length=3)

Perhaps `will` be used?

Reply:
> Perhaps will be used?

I intentionally used "can" because we might never use it. It all depends.

Reply:
In that case, I think `might` sounds better. But it's ok to leave as is if you prefer.